src/Cake.GitHub/GitHubAppAliases.Issues.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Cake.Core;
+using Cake.Core.Annotations;
+using Octokit;
+using System.Threading.Tasks;
+
+namespace Cake.GitHub
+{
+    [CakeAliasCategory("GitHub")]
+    public static partial class GitHubAliases
+    {
+        /// <summary>
+        /// Sets the Milestone for an Issue or Pull Request.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="userName">The user name to use for authentication (pass <c>null</c> when using an access token).</param>
+        /// <param name="apiToken">The access token or password to use for authentication.</param>
+        /// <param name="owner">The owner (user or group) of the repository.</param>
+        /// <param name="repository">The name of the repository.</param>
+        /// <param name="number">The number of the Issue or Pull Request to set the Milestone for.</param>
+        /// <param name="milestoneTitle">
+        /// The title of the milestone to assign the Issue or Pull Request to.
+        /// Note that GitHub treats Milestone titles case-sensitive.
+        /// </param>
+        /// <param name="settings">Additional settings for updating the Milestone (optional).</param>
+        [CakeMethodAlias]
+        public static async Task GitHubSetMilestoneAsync(
+            this ICakeContext context,
+            string? userName,
+            string apiToken,
+            string owner,
+            string repository,
+            int number,
+            string milestoneTitle,
+            GitHubSetMilestoneSettings? settings = null)
+        {
+            settings = settings ?? new GitHubSetMilestoneSettings();
+
+            var connection = CreateConnection(userName, apiToken);
+            var githubClient = new GitHubClient(connection);
```

Comment:
should be gitHubClient if following the correct naming convention, but this is really, really, really low priority :)

Reply:
Fixed with 7703cb4c6a575b4baac1105a76a868ed43d5b52e